### PR TITLE
fix Err.Timestamp

### PR DESCRIPTION
**Pull request by bjdgyc**
fix Err.Timestamp

### CODE REVIEW

**Comment by wheelcomplex** [comment]
fix: https://github.com/RobotsAndPencils/buford/issues/37

File: push/service_test.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func TestTimestampError(t *testing.T) {
 
 	handler.HandleFunc("/3/device/", func(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusGone)
-		w.Write([]byte(`{"reason":"Unregistered","timestamp":12622780800}`))
+		w.Write([]byte(`{"reason":"Unregistered","timestamp":1262278080000}`))
```
Comment (nathany) [inline push/service_test.go]: shouldn't this have one more zero?

and then line 108/109 wouldn't need to change.

**Comment by nathany** [comment]
Thanks for submitting a pull request.

File: push/service_test.go (hunk 1 above)
Comment (bjdgyc) [inline push/service_test.go]: The true data is `1262278080000`,because,it is Millisecond.
这样改的原因是与实际的数据一致，3q

Comment (nathany) [inline push/service_test.go]: 12622780800 seconds in milliseconds = 12622780800000

you're missing a zero.

Comment (bjdgyc) [inline push/service_test.go]: 1262278080  is 2010/1/1 0:48:0
12622780800 is null
the unixtime is error.

Comment (nathany) [inline push/service_test.go]: I get http://play.golang.org/p/ctPrCvcsWV
2370-01-01 00:00:00 +0000 UTC
2009-12-31 16:48:00 +0000 UTC

But I guess it doesn't matter either way, as this is just the test.

**Comment by nathany** [comment]
merged. thanks again!

File: push/service_test.go (hunk 1 above)
Comment (bjdgyc) [inline push/service_test.go]: ok,3q.i think you are right

**Pull request closed without merging**